kolibri/core/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+"""TODO: Write something about this module (everything in the docstring
+enters the docs)
+
+.. moduleauthor:: Learning Equality <info@learningequality.org>
```

Comment:
True that

I was thinking about making the `__init__.py` files into templates. I'm not sure if the `from __future__` stuff should be three lines. I'll put on the TODO to make the perfect header for a file.

Also considering to remove the utf-8 header thing... after all, it's sooooooo rare that you need a utf-8 character in source code, and if you do, then stuff will break at compile time, not runtime, so it's easy to spot. Only Python 2 needs it.

A shorter version would be like this:

```
"""
This is a nice module <3
"""
from __future__ import absolute_import, print_function, unicode_literals
```